src/routes/JobsRoute.js

```diff
@@ -98,13 +97,8 @@ class JobsRoute extends React.Component {
         history.push(`/local-kb-admin/${record.id}${location.search}`);
       }
     }
-
-    const prevDeletedJobId = get(prevProps, 'location.state.deletedJobId', '');
-    const currentDeletedJobId = get(this.props, 'location.state.deletedJobId', '');
-    if (prevDeletedJobId !== currentDeletedJobId) {
-      const name = get(this.props, 'location.state.deletedJobName', '');
-      if (name !== '') this.showToast('ui-local-kb-admin.job.delete.success', 'success', { name });
-    }
+    calloutLogic(this.props, prevProps, 'deleted', this.callout.current.sendCallout);
+    calloutLogic(this.props, prevProps, 'created', this.callout.current.sendCallout);
```

Comment:
These 2 line here seem to be unclear, as its just calling 2 callouts back to back but only one of them would go through. 

If it makes things easier add a new `created` vs `deleted` boolean to the location state and conditionally call the `calloutLogic` function

Reply:
Done